packages/now-next/src/index.ts

```diff
@@ -569,13 +569,13 @@ export const build = async ({
         },
 
         // error handling
-        ...(output['404']
+        ...(output[path.join(entryDirectory, '404')]
```

Comment:
Should this be this to be safe?
```suggestion
        ...(output[path.join('./', entryDirectory, '404')]
```

Reply:
Outputs don't have starting slash

Reply:
The above code would remove it if present:
```js
> path.join('./', '/abc')
'abc'
```

Reply:
Ah yeah that works, updated 👍